service/src/test/java/uk/gov/hmcts/reform/fpl/service/DocumentGeneratorServiceTest.java

```diff
@@ -30,16 +31,18 @@ void testEmptyCaseDetailsSuccessfullyReturnsByteArray() throws IOException {
 
         String content = textContentOf(documentGeneratorService.generateSubmittedFormPDF(caseDetails));
 
-        assertThat(content).contains("Case ID: 123");
+        assertThat(content).contains("C110A");
```

Comment:
the idea of this was to be that the whole pdf would be tested in here as a completed form...

Reply:
So this is now redundant?

Reply:
Whole PDF is tested in test below.

Reply:
Can we remove this test Kurt?